Update build-image-with-packer.md

boumenot opened the pull request:
It is generally not sufficient to call sysprep, and then exit.  To ensure a reliable build, one should wait for sysprep to complete before returning from the provisioner.  I updated the document to match the advice I give in the Azure examples, and in Packer's own documentation.

azurecla commented:
Hi __@jierong__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jierong). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cjgronlund commented:
@sidneyh 
Please check this change, and let me know whether to accept the PR.
Thanks!

jierong commented:
I think we can use Get-AzureRmSqlDatabaseActivity to replace Get-AzureRmSqlElasticPoolDatabaseActivity

tysonn commented:
@sidneyh - Can you please review this contribution?

sidneyh commented:
#sign-off

cjgronlund commented:
@sidneyh 
This PR appears to be a dupe of https://github.com/Azure/azure-content/pull/6050. Please tell me specifically what you'd like me to do this this PR (6051). Close? Thanks!

sidneyh commented:
Hi, you can close it then. Thanks!

cjgronlund commented:
Duplicate of https://github.com/Azure/azure-content/pull/6050. Closing.

boumenot committed:
Update build-image-with-packer.md

It is generally not sufficient to call sysprep, and then exit.  To ensure a reliable build, one should wait for sysprep to complete before returning from the provisioner.  I updated the document to match the advice I give in the Azure examples, and in Packer's own documentation.
